doc/api/esm.md

```diff
@@ -1152,10 +1152,11 @@ The `conditions` property on the `context` is an array of conditions for
 for looking up conditional mappings elsewhere or to modify the list when calling
 the default resolution logic.
 
-The [current set of Node.js default conditions][Conditional Exports] will always
-be in the `context.conditions` list passed to the hook. If the hook wants to
-ensure Node.js-compatible resolution logic, all items from this default
-condition list **must** be passed through to the `defaultResolve` function.
+The current [package exports conditions][Conditional Exports] will always be in
+the `context.conditions` array passed into the hook. To guarantee _default
+Node.js module specifier resolution behavior_ when calling `defaultResolve`, the
+`context.conditions` array passed to it **must** include **all** elements of the
```

Comment:
Maybe just make `must include all` bold entirely?

Reply:
I wonder if we can turn this portion of the documentation into more of a specification. Others who have some experience w/ this in the past might wish to weigh in here. There are several choices. It could be in all caps too.

Refs: https://tools.ietf.org/html/rfc7230#section-3.5
Refs: https://tools.ietf.org/html/draft-ietf-dispatch-javascript-mjs-05

/cc @bmeck, @MylesBorins, @ljharb

Reply:
For example:

>   In the interest of robustness, a server that is expecting to receive
>   and parse a request-line SHOULD ignore at least one empty line (CRLF)
>   received prior to the request-line.

Refs: https://tools.ietf.org/html/rfc7230#section-3.5

Reply:
The above is wrong because it mixes two different ways of adding emphasis. This is a Markdown document after all.

```suggestion
`context.conditions` array passed to it _**must**_ include _all elements_ of the
```